### ISSUE TITLE
Error with missing passport-config.json; no documentation

### ISSUE THREAD

**Issue by rjaguar3** [mostafab/neg5#76]
When attempting to run a development server, I get the following error:
```
/home/greg/programming/neg5/app/build/config/passport/config.js:26
    throw err;
    ^

Error: ENOENT: no such file or directory, open '/home/greg/programming/passport-config.json'
    at Error (native)
    at Object.fs.openSync (fs.js:549:18)
    at Object.fs.readFileSync (fs.js:397:15)
    at Object.<anonymous> (/home/greg/programming/neg5/app/build/config/passport/config.js:23:27)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
```
I couldn't find any instructions on how to create passport-config.json.

**Comment by jonahgreenthal** [comment]
Is your fork up-to-date? I had this same problem, but @mostafab fixed it (for me, at least) in f57f562 and 8d562d9.

**Comment by rjaguar3** [comment]
Works for me now; thanks Jonah.
